src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -1,12 +1,13 @@
 package mesosphere.marathon
 
-import mesosphere.marathon.tasks.IterativeOfferMatcherConfig
+import mesosphere.marathon.tasks.{ OfferReviverConf, IterativeOfferMatcherConfig }
 import org.rogach.scallop.ScallopConf
 import scala.sys.SystemProperties
 
 import mesosphere.marathon.io.storage.StorageProvider
 
-trait MarathonConf extends ScallopConf with ZookeeperConf with IterativeOfferMatcherConfig with LeaderProxyConf {
+trait MarathonConf extends ScallopConf with ZookeeperConf with IterativeOfferMatcherConfig with LeaderProxyConf
```

Comment:
Nit: one of these configuration traits is not named the same as the others, but it was not created nor touched in this PR.